test/test_simulation_solution.jl

```diff
@@ -442,17 +442,17 @@ function test_load_simulation(file_path::String)
                     name,
                     24,
                 ]
-            cache = collect(values(sim_cache.internal.simulation_cache))[1].value[name]
+            cache = sim_cache.internal.simulation_cache[PSI.CacheKey(PSI.TimeStatusChange, PSY.ThermalStandard)].value[name]
```

Comment:
there is a function to retrieve the cache here it is cleaner.

Reply:
ok , I'll make that change